src/t/d11.clj

```diff
@@ -0,0 +1,88 @@
+(ns t.d11
+  (:gen-class)
+  (:require
+    [clojure.string :as cljstr]
+    [clojure.set :as cljset]
+    [clojure.core.match :refer [match]]))
+
+(def sample
+"125 17")
+
+(defn parse-input
+  [input]
+  (->> input
+       (re-seq #"\d+")
+       (map parse-long)
+  ))
+
+(defn split-stone-2
+  [stone]
+  (let [str-stone (str stone)
+        mid-i (quot (count str-stone) 2)]
+    (->>
+      [(take mid-i str-stone) (drop mid-i str-stone)]
+      (map (fn [l] (apply str l)))
+      (map parse-long))))
```

Comment:
Are you aware of the [subs](https://clojuredocs.org/clojure.core/subs) core library function?

Reply:
I read your code, now I am >.>
but see, I am using `take` and `drop` so there is a little bit of progress :D